ci/check-style.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+if [ "${CHECK_STYLE}" != "yes" ]; then
+    echo "Skipping code style check as it is disabled for this build."
+    exit 0
+fi
+
+# This script assumes it is running from /var/tmp/build as setup by the
+# Dockerfile.
+cp -r gccpp gccpp-orig
+cd gccpp
+find . \( -path ./.git -prune -o -path ./third_party -prune \) \
```

Comment:
Can we make this command a CMake target, or use this in the example in the README (assuming it works):

`find bigtable  -o -name '*.h' -o -name '*.cc' -print0 | xargs -0 clang-format -i`

Right now, the README is hardcoding `bigtable/client/*.{h,cc}` which will stop working as soon as we add a subdirectory under `bigtable`.

Reply:
Better example, thanks!  Done.